config/apis.ts

```diff
@@ -6,6 +6,30 @@ const addressValidation: KeyAuthAPI = {
   name: 'Address Validation API',
 };
 
+const appealsStatus: KeyAuthAPI = {
+  acl: 'appeals',
+  key: 'appeals',
+  name: 'Appeals Status API',
+};
+
+const clinicalHealth: OAuthAPI = {
+  authzEndpoint: process.env.AUTHZ_SERVER_CLINICAL_FHIR,
+  key: 'clinicalHealth',
+  name: 'Clinical Health API (FHIR)',
+};
+
+const decisionReviews: KeyAuthAPI = {
+  acl: 'appeals',
```

Comment:
acl will be `hlr` as it is a separate API from the API currently using the `appeals` acl

Reply:
okie dokie I'll change the decisionReviews acl

Reply:
updated in latestCommit